siege_utilities/databricks/lakehouse_federation.py

```
@@ -0,0 +1,108 @@
+    ]
+    for table in tables:
+        statements.append(
+            build_foreign_table_sql(
+                catalog=catalog,
+                schema=schema,
+                table=table,
+                connection_name=connection_name,
+                source_schema=source_schema,
+                source_table=table,
+            )
+        )
```

Ruff (0.15.13)

[warning] Use `list.extend` to create a transformed list

(PERF401)